<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessGUI</class>
 <widget class="QMainWindow" name="ProcessGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ARCH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Process</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pray_meme">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProcessGUI</receiver>
   <slot>RefreshProcesses()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProcessGUI</receiver>
   <slot>FindProcess(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ProcessGUI</receiver>
   <slot>SelectProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>RefreshProcesses()</slot>
  <slot>FindProcess(QString)</slot>
  <slot>SelectProcess()</slot>
 </slots>
</ui>
